core/domain/search_services.py

```diff
@@ -154,7 +154,7 @@ def _should_index_collection(collection):
     """Checks if a particular collection should be indexed.
 
     Args:
-        collection: CollectionSummaryModel.
+        collection: CollectionSummaryModel. Collection summary model object.
```

Comment:
The collection summary model to check.

(Btw, please fix this throughout your PR. Docstrings that just restate the typeinfo are not very useful -- provide the context instead.)